<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrainPreview</class>
 <widget class="QWidget" name="GrainPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>175</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>175</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="grainPreviewGraph" name="tabFace">
      <attribute name="title">
       <string>Face</string>
      </attribute>
     </widget>
     <widget class="grainPreviewGraph" name="tabRegression">
      <attribute name="title">
       <string>Regression</string>
      </attribute>
     </widget>
     <widget class="grainPreviewGraph" name="tabAreaGraph">
      <attribute name="title">
       <string>Area Graph</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>grainPreviewGraph</class>
   <extends>QWidget</extends>
   <header location="global">uilib</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
